backend/inspirehep/config.py

```diff
@@ -151,7 +151,7 @@ def annotate(self, task):
 #: It should be changed before deploying.
 MAX_CONTENT_LENGTH = 100 * 1024 * 1024  # 100 MiB
 #: Max upload size for form data via application/mulitpart-formdata.
-SESSION_COOKIE_SECURE = True
+SESSION_COOKIE_SECURE = False
```

Comment:
Can you make sure this is set to `True` in kubernetes repo?

Reply:
Looks like it's not

Reply:
Then you can create a PR there first.

https://gitlab.cern.ch/inspire/kubernetes/blob/master/hep/base/config/inspirehep.cfg